scripts/build_utils.sh

```diff
@@ -489,6 +489,10 @@ echo "deb [arch=amd64] http://us.archive.ubuntu.com/ubuntu/ $DIST-backports main
 echo "deb [arch=amd64] http://security.ubuntu.com/ubuntu $DIST-security main restricted universe multiverse" >> /etc/apt/sources.list
 env DEBIAN_FRONTEND=noninteractive apt-get update -y -o Acquire::Languages=none -o Acquire::Translation=none || true
 env DEBIAN_FRONTEND=noninteractive apt-get install -y gnupg
+env DEBIAN_FRONTEND=noninteractive apt-get install -y -V ca-certificates lsb-release wget
+wget https://apache.bintray.com/arrow/$(lsb_release --id --short | tr 'A-Z' 'a-z')/apache-arrow-archive-keyring-latest-$(lsb_release --codename --short).deb
```

Comment:
currently, we don't have debian building hosts. please check the compatibility of dist[1] with ubuntu focal and use the one compatible with ubuntu focal.

instead of download and install deb packages, i'd suggest just add a source list under `/etc/apt/sources.list.d/arrow.list`, like
```
deb https://apache.jfrog.io/artifactory/arrow/debian bulleye main
```

in this case, i assume that bullseye is compatible with ubuntu/focal.

---

[1] https://apache.jfrog.io/artifactory/arrow/debian/dists/

Reply:
ok.
will try that (will use a container for testing this)

for installing apache-arrow I used the following recommendations https://arrow.apache.org/install/

Reply:
had tried that 
getting the following
Hit:1 http://archive.ubuntu.com/ubuntu bionic InRelease
Hit:2 http://archive.ubuntu.com/ubuntu bionic-updates InRelease                                                          
Hit:3 http://security.ubuntu.com/ubuntu bionic-security InRelease                                                        
Hit:4 http://archive.ubuntu.com/ubuntu bionic-backports InRelease                                                        
Ign:5 https://apache.jfrog.io/artifactory/arrow/debian bulleye InRelease                           
Err:6 https://apache.jfrog.io/artifactory/arrow/debian bulleye Release
  404  Not Found [IP: 54.148.141.177 443]
Reading package lists... Done
E: The repository 'https://apache.jfrog.io/artifactory/arrow/debian bulleye Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.

Reply:
the following is working (container ubuntu-image)
sudo apt update
sudo apt install -y -V ca-certificates lsb-release wget
wget https://apache.jfrog.io/artifactory/arrow/ubuntu/apache-arrow-apt-source-latest-bionic.deb
sudo apt install -y -V ./apache-arrow-apt-source-latest-bionic.deb
sudo apt update

Reply:
i still prefer install packages from repo, instead of downloading them:

```sh
curl -L https://dist.apache.org/repos/dist/dev/arrow/KEYS | sudo apt-key add -
add-apt-repository "deb [arch=amd64] http://dl.bintray.com/apache/arrow/ubuntu $DISTRO main"
```

Reply:
ok, testing that on container-ubuntu-image

- the curl should be installed (I imagine it is)
- add-apt-repository command should be installed 
- gnupg1 should be installed